test/LEGO.AsyncAPI.Tests/Serialization/AsyncApiYamlWriterTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) The LEGO Group. All rights reserved.
+
+namespace LEGO.AsyncAPI.Tests.Writers
+{
+    using LEGO.AsyncAPI.Writers;
+    using NUnit.Framework;
+    using System;
+    using System.IO;
+
+    internal class AsyncApiYamlWriterTests : TestBase
```

Comment:
Here is the tests you asked to bring back. Before they were part of that static helper class but the logic was moved into the writer